CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To publish a new release run `scripts/release` from the project directory.
 
 ## [Unreleased]
 
+### Added
+- Support for WezTerm - https://wezfurlong.org/wezterm/
```

Comment:
Please do add a link to this pull request (see past changelogs for examples), and make `WezTerm` a link instead of adding the URL separately, i.e.

```md
- Support for [WezTerm](https://wezfurlong.org/wezterm/) (see [GH-182]).

[GH-182]: https://github.com/lunaryorn/mdcat/pull/182
```